tickets-test/src/main/java/no/unit/nva/publication/ticket/test/TicketTestUtils.java

```diff
@@ -394,8 +394,8 @@ private static void publishFiles(Publication publication) {
         publication.setAssociatedArtifacts(new AssociatedArtifactList(list));
     }
 
-    private static Publication randomPublicationWithStatusAndDoi(PublicationStatus status) {
-        return PublicationGenerator.randomPublication().copy()
+    private static Publication publicationWithStatusAndDoi(PublicationStatus status) {
+        return PublicationGenerator.randomPublication(AcademicArticle.class).copy()
```

Comment:
Degree that causes problems?
Can we use randomNonDegreePublication()

Reply:
In general, I am not sure if we want randomness here. We should however check every type where it has some meaning to do so. If you feel strongly about this, we should exhaustively check the randomness so we don't introduce unnecessary flakiness.